docs/contribute-guide/markdown.md

```diff
@@ -0,0 +1,138 @@
+# 让你提交的 Markdown Pro 起来
+
+> Markdown 也需要规范？
+
+是的。虽然我们往往将 Markdown 作为 HTML 生成语言使用，但 Markdown 设计之初就希望保证其代码在纯文本界面下的可读性。你的控制台、编辑器不会优化文字的排版（一般都会用等宽字体），但是浏览器会，因此就算你的 Markdown 写的非常的混乱，在浏览器（或者 ~~通过浏览器套壳实现的~~ 所见即所得编辑器，比如 Typora）里看起来依旧非常正常。为了照顾 ~~强迫症患者~~ 这种需求，我们需要给 Markdown 一个规范。
+
+## 中文文案的通用规范
+
+详情请参考[《中文文案排版指北》](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-Hans.md)，下面仅指出两个重点。
+
+### 盘古之白
+
+中文和西文、数字之间加空格。按照 [BV13M4y157Xc](https://www.bilibili.com/video/BV13M4y157Xc) 中的说法，中西文之间的间隔应该是全角的四分之一宽，但是四分之一空格既不方便打也不方便用等宽字体显示，所以统一使用普通空格代替。（按照某位不知名人士的说法，聪明的浏览器会忽略这个空格，愚蠢的终端需要这个空格）
+
+```txt
+哦，我的天呐，你居然连 Markdown 是什么都不知道，真是见鬼！
+我真想用隔壁苏珊大妈 20" 的靴子狠狠的踢你的屁股，你这个坏东西！
+```
+
+需要注意的是，如果西文后面直接接符号，则不需要空格。
+
+```txt
+我们的代码托管在 GitHub。
+```
+
+### 全角标点
+
+除非是西文整句，否则全部使用中文标点。
+
+```txt
+俗话说得好，“Too young, too simple.”
```

Comment:
I might not appreciate this example, because the quotation marks both in standard simplified Chinese and English are the same; thus, it might cause ambiguity. It would be better if this was pointed out in the documentation.

Reply:
"" != “”

Reply:
> "" != “”

Your perception is wrong. The marks `“`, `”` and so on are not special punctuation marks in Chinese, and they are originated from English. `"` is used for convenience, but not in standard.

Reply:
How about:

```txt
俗话说得好，「Too young, too simple.」
```